AirCasting/SessionViews/ShareSession/SharePage.swift

```diff
@@ -0,0 +1,58 @@
+// Created by Lunar on 08/07/2021.
+//
+import MessageUI
+import SwiftUI
+
+struct ActivityViewController: UIViewControllerRepresentable {
+    typealias Callback = (_ activityType: UIActivity.ActivityType?, _ completed: Bool, _ returnedItems: [Any]?, _ error: Error?) -> Void
+    
+    let sharingFileisLocationless: Bool
```

Comment:
I think this generally apply to sharing file instead of a link, so no need to mention the locationless here